x-pack/plugins/security_solution/public/common/components/exceptions/builder/index.test.tsx

```diff
@@ -0,0 +1,418 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { ThemeProvider } from 'styled-components';
+import { mount } from 'enzyme';
+import euiLightVars from '@elastic/eui/dist/eui_theme_light.json';
+import { wait as waitFor } from '@testing-library/react';
+
+import {
+  fields,
+  getField,
+} from '../../../../../../../../src/plugins/data/common/index_patterns/fields/fields.mocks.ts';
+import { getExceptionListItemSchemaMock } from '../../../../../../lists/common/schemas/response/exception_list_item_schema.mock';
+import { getEntryMatchAnyMock } from '../../../../../../lists/common/schemas/types/entry_match_any.mock';
+
+import { useKibana } from '../../../../common/lib/kibana';
+import { getEmptyValue } from '../../empty_value';
+
+import { ExceptionBuilderComponent } from './';
+
+jest.mock('../../../../common/lib/kibana');
+
+describe('ExceptionBuilderComponent', () => {
+  const getValueSuggestionsMock = jest.fn().mockResolvedValue(['value 1', 'value 2']);
+
+  beforeAll(() => {
```

Comment:
Nit: simpler to make this a `beforeEach` and create `getValueSuggestionsMock` each time so that you're not carrying state between tests (via this mock). You very rarely need `afterEach` and `beforeAll/afterAll`.

Reply:
Updated!